### PR TITLE
Tweaks to the Home Page - Mobile

### PR DESCRIPTION
**Pull request by mmacedom**
header logo adjusted top, Hero black box opacity change, Teaching H1/Div padding

### CODE REVIEW

File: src/components/RenderRouter/HeroItem.scss [hunk 1]
```diff
@@ -324,7 +324,7 @@
 
     .heroBlackBox {
         position: absolute;
-        background-color: #000000aa;
+        background-color: rgba(0,0,0,1);
```
Comment (GeorgeBellTMH) [inline src/components/RenderRouter/HeroItem.scss]: I think we want to keep the alpha on mobile....there was some discussions and the mobile design wasn't updated... @deeinstereo to confirm....

Comment (deeinstereo) [reply src/components/RenderRouter/HeroItem.scss]: @GeorgeBellTMH sorry can you clarify, not sure I understand what you're referring to.

Comment (mmacedom) [reply src/components/RenderRouter/HeroItem.scss]: @deeinstereo @georgevicbell , he is referring to the black box behind the hero copy on the top of home page (mobile). We initially had it semi transparent and then last night I modified it to full opacity (black) to match the figma mocks. Let me know how you guys would prefer to keep that.

Comment (deeinstereo) [reply src/components/RenderRouter/HeroItem.scss]: @mmacedom thanks for clarifying! I prefer full black if it works but I know we've been struggling with that and sometimes some opacity is better. I'm good with whatever is the strongest option.

Comment (mmacedom) [reply src/components/RenderRouter/HeroItem.scss]: @georgevicbell @deeinstereo okay. Either way, it's a very simple change to the code. So it's not a big deal to change it later, if we have to.

Comment (GeorgeBellTMH) [reply src/components/RenderRouter/HeroItem.scss]: Going with all black in mobile for now.

**Pull request merged**